README.md

```diff
@@ -43,10 +45,33 @@ Exec type options:<br />
 The "Settings" window on a mod displays the Command Line to play that mod directly via Quick Launch. You can copy that Quick Launch Command Line and use it to create a shortcut to easily launch that mod. <br />
 
 <br /><br />
-## **Software Rendering Mode:**<br />
-By default KnossosNET will render the UI in the GPU, can be set to run completely on software rendering what effectively avoids any use of the user GPU, this will come at the cost of increased CPU usage, what should not be a problem when in idle.
+## **Tool Quick Launch:**<br />
+Just like mods QuickLaunch, you can launch installed mod development tools with a cmdline argument. Windows users have the possibility to create a shortcut to desktop with a click. This may be extended to Linux and Mac later on.<br />
```

Comment:
Suggest changing to 
`You can launch installed mod development tools with a command line argument (just like how you can launch mods with QuickLaunch).